docs/quickstart/pods/gpu-pod.yaml

```diff
@@ -12,7 +12,8 @@ spec:
   containers:
     - name: main
       image: ubuntu
-      args: ["sleep", "infinity"]
+      command: ["bash", "-c"]
+      args: ["nvidia-smi; trap 'exit 0' TERM; sleep 9000 & wait"]
```

Comment:
What do you think about making it `infinity` instead of `9000`?
We prefer not to make time limit so people can run it go do other things and get back directly where they left.

Reply:
For quick testing, i think people may just want to run a simple container like this quickly and get onto more useful things once they get a hang of it. I would prefer to increase the time to a bigger number than infinity because i had to force terminate this pod.

Reply:
Yeah you had to force terminate your code because `sleep` does not handle signals, no matter if its 9000 or infinity.
You handled it with the `trap` command (which is a great option), and I suggest of keeping the trap, just changing the number to infinity.
So you wont need to force terminate the pod + the guide will not be depend on times.
What do you think?

Reply:
Okay i will update it :)